tests/unittests/PartitionerTest.cpp

```diff
@@ -19,6 +19,10 @@
 
 #include "gtest/gtest.h"
 
+#ifndef GLOW_DATA_PATH
```

Comment:
What is this?

Reply:
this is required for the fbcode setup. Man, can you add a comment on this.